Rules/CommonScripts/ChatCommands.as

```diff
@@ -212,7 +212,6 @@ bool onServerProcessChat(CRules@ this, const string& in text_in, string& out tex
 			stone.server_SetQuantity(500);
 			//gold
 			CBlob@ gold = server_CreateBlob('mat_gold', -1, pos);
```

Comment:
The blob doesn't need to be stored to a variable since you have removed the line below it.
```suggestion
			server_CreateBlob('mat_gold', -1, pos);
```

Reply:
You forgot this change.